### PR TITLE
Update deploying.asciidoc

### PR DESCRIPTION
**Pull request by meons**
Be careful copy/paste ^^

### CODE REVIEW

**Comment by meons** [comment]
Could someone merge this PR ?

File: docs/static/deploying.asciidoc [hunk 1]
```diff
@@ -158,7 +158,7 @@ Logstash can expose endpoint listeners with the respective
 <<plugins-inputs-http,HTTP>> input plugins. The data sources enumerated below
 are typically ingested through one of these three protocols.
 
-NOTE: The TCP protocol does not support application-level acknowledgements, so
+NOTE: The UDP protocol does not support application-level acknowledgements, so
```
Comment (jsvd) [inline docs/static/deploying.asciidoc]:
```suggestion
NOTE: The TCP and UDP protocols do not support application-level acknowledgements, so
```

**Comment by jsvd** [comment]
closed by https://github.com/elastic/logstash/commit/28345e8caf961865bd01c459f8f55060f5d8a9cc

**Comment by jsvd** [comment]
backported to 7.x as 8de6cecc3

**Pull request closed without merging**